packages/lib/src/types/global-types.ts

```diff
@@ -298,6 +298,10 @@ export interface PaymentData extends PaymentMethodData {
     clientStateDataIndicator: boolean;
     sessionData?: string;
     storePaymentMethod?: boolean;
+    billingAddress?: AddressData;
+    deliveryAddress?: AddressData;
+    shopperEmail?: string;
+    shopperName?: string;
```

Comment:
Can we actually return `shopperEmail` and `shopperName` here? Which payment method does that?

Reply:
I think I have a new idea how to fix in a proper way. Let me try some typescript ideas.